kuma/dashboards/forms.py

```diff
@@ -13,6 +13,14 @@
     ('week', _('Week')),
     ('month', _('30 days')),
 ]
+ALL_AUTHORS = 0
+KNOWN_AUTHORS = 1
+UNKNOWN_AUTHORS = 2
+AUTHOR_CHOICES = [
+    (ALL_AUTHORS, _('All Authors')),
+    (KNOWN_AUTHORS, _('Known Authors')),
+    (UNKNOWN_AUTHORS, _('Unknown Authors')),
+]
 
```

Comment:
This seems to be a duplicate of the same data in RevisionDashboardForm. I think this one is the one to remove.